distributor-widget/integrations.md

```diff
@@ -78,6 +78,53 @@ Be sure to also enable Enhanced Ecommerce in you Google Analytics under Admin &g
 
 When you have Mews Merchant set up, a payment by a customer is legally required to happen on our domain. Therefore, all the transactions during a checkout are attributed to Mews domain. This is an unfortunate limitation of the checkout process that we cannot currently overcome.
 
+### Conditionally firing tags based on tracking consents
+
+Google Tag Manager integrations are very often used to track users, be it with cookies or other approaches. You can use [TrackingConsents](integrations.md#trackingconsents) and Google Tag Manager to use integrations only when permitted.
+
+Commonly used integration which often needs a tracking consent is Universal Analytics.
+
+Let's see how you can fire Universal Analytics tag only when [TrackingConsents](integrations.md#trackingconsents) permit it.
+
+0. Set up Universal Analytics in Google Tag Manager:
```

Comment:
@zdrazil It shouldn't start with `0`, it should start with `1`.

Reply:
It's zero, because it's not described in this section. Do you still prefer starting with zero?

Reply:
changed it to 1